### ISSUE TITLE
Accesible colors

### ISSUE THREAD

**Issue by btzr-io** [btzr-io/Villain#149]
I like the colors but I'm not sure if they are good for accessibility see: https://accessible-colors.com
- Add any tools for this ?
- Update colors of light theme to match best / close `accesible-color`.
- Update colors of darkt theme to match best / close `accesible-color`.
- Update example demo colors theme to match best / close `accesible-color`.

**Comment by shingz96** [comment]
you can use the Contrast color with YIQ formula

**Comment by lucasjs** [comment]
@btzr-io Hey, may I help you?
I'm helping some open source projects with that issue (you can see some pull requests on my profile).

Let me know if you need me. :)

**Comment by lucasjs** [comment]
@btzr-io After checking with a few validators, all colors are fine.
But, a few `aria-label`'s are missing in the buttons.

I could help with that too. What's better, change this issue or create a new one?
I'm already working on that.

**Comment by btzr-io** [comment]
@lucasjs Feel free to create a new one, but I think is part of #84

**Comment by lucasjs** [comment]
@btzr-io that's nice!

<img width="979" alt="Screen Shot 2019-10-29 at 19 14 31" src="https://user-images.githubusercontent.com/14945222/67813476-617f4780-fa80-11e9-95da-1d681e9ec04c.png">
<img width="729" alt="Screen Shot 2019-10-29 at 19 15 30" src="https://user-images.githubusercontent.com/14945222/67813523-7b208f00-fa80-11e9-8dcf-b20233b1bc3a.png">

Then, all colors are ok. :)
Great job!

**Comment by btzr-io** [comment]
@lucasjs Wow that looks interesting what is that tool (validator) ?

**Comment by btzr-io** [comment]
@lucasjs Do you know any tools like that one, to integrate on the project ?

**Comment by lucasjs** [comment]
@btzr-io I do! In a few days I'll come back on this issue. :)

**Comment by btzr-io** [comment]
@lucasjs  ok, thanks :v:
